webui/react/src/components/UserSettings.tsx

```diff
@@ -346,33 +348,37 @@ const UserSettings: React.FC<Props> = ({ show, onClose }: Props) => {
               </div>
             </Section>
             <Section title="Advanced">
-              <Body>
-                Advanced features are potentially dangerous and could require you to completely
-                reset your user settings if you make a mistake.
-              </Body>
-              <Accordion title="I know what I'm doing">
-                <Space>
-                  <Button
-                    danger
-                    type="primary"
-                    onClick={() =>
-                      confirm({
-                        content:
-                          'Are you sure you want to reset all user settings to their default values?',
-                        onConfirm: () => {
-                          setMode(Mode.System);
-                          userSettings.clear();
-                        },
-                        onError: handleError,
-                        title: 'Reset User Settings',
-                      })
-                    }>
-                    Reset to Default
-                  </Button>
-                  <Button onClick={() => UserSettingsModal.open()}>Edit Raw Settings (JSON)</Button>
-                  <UserSettingsModal.Component />
-                </Space>
-              </Accordion>
+              <Grid gap={ShirtSize.Medium} minItemWidth={400}>
```

Comment:
We could create a separate ticket for replacing all of them (or there might be one already), but I think we want to use hew `Collection` instead of determined's `Grid`.

But also, since this is only vertical, would prefer hew `Column` anyway.

Reply:
created ticket for replacing `Grid` replacement with `Collection`